FirebaseAppDistribution/CHANGELOG.md

```diff
@@ -10,15 +10,15 @@
 - [changed] Sign out the Tester when the call to fetch releases fails with an unauthenticated error.
 - [fixed] Crash caused by trying to parse response as JSON when response is nil (#6996).
 
-# v0.9.3
+# v0.9.3-beta
```

Comment:
Hmm, here and below this tag doesn't actually exist (it was `AppDistribution-0.9.3`). I think it's reasonable to remove the `beta` and just leave it as is since the `0` indicates beta already.

Reply:
Done in [60b3089](https://github.com/firebase/firebase-ios-sdk/pull/9777/commits/60b30895f4e5b2d68dda1657dba38903730bfe4f)